src/attribution/onINP.ts

```diff
@@ -239,10 +239,16 @@ const getIntersectingLoAFs = (
 
 const attributeINP = (metric: INPMetric): INPMetricWithAttribution => {
   const firstEntry = metric.entries[0];
+  const inpTime = firstEntry.startTime + metric.value;
   const group = entryToEntriesGroupMap.get(firstEntry)!;
 
   const processingStart = firstEntry.processingStart;
-  const processingEnd = group.processingEnd;
+  // processingEnd can extend beyond duration for modals where we artificially
+  // mark event timing duration as that paint.
```

Comment:
Nit: "as that paint" comment is confusing (there is no actual paint)

Perhaps just "where duration ends at the time modal dialog started (and provided visual feedback)"

Reply:
Updated.